src/torchjd/autojac/_transform/_differentiation.py

```diff
@@ -3,8 +3,8 @@
 
 from torch import Tensor
 
-from torchjd._transform._utils import ordered_set
-from torchjd._transform.base import _A, Transform
+from torchjd.autojac._transform._utils import ordered_set
+from torchjd.autojac._transform.base import _A, Transform
```

Comment:
relative import?

Reply:
I think that absolute gives a warning right ? I had opted for relative when I broke everything up.